### PR TITLE
Increments build number, fix provisioning profiles

### PR DESCRIPTION
**Pull request by DanWebster**

### CODE REVIEW

**Comment by DanWebster** [comment]
Also had to change the bundleID on the biomarin.pku (removed .study) because the two bundleIDs were identical otherwise with both of them having .study appended

File: BiomarinPKU/BiomarinPKU.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -991,7 +991,8 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_ENTITLEMENTS = BiomarinPKU/BiomarinPKU.entitlements;
-				CODE_SIGN_STYLE = Manual;
+				CODE_SIGN_IDENTITY = "iPhone Developer";
+				CODE_SIGN_STYLE = Automatic;
```
Comment (dephillipsmichael) [inline BiomarinPKU/BiomarinPKU.xcodeproj/project.pbxproj]: I think we tend to do manual with a matching qualifier.  Automatic can work as well, but it doesn't work well for people like me who don't have sufficient permissions to create/edit provisioning profiles.

Comment (dephillipsmichael) [reply BiomarinPKU/BiomarinPKU.xcodeproj/project.pbxproj]: Xcode probably updated something in the provisioning profile for you to get it to work, so you probably should be able to switch back to what it was before.

Comment (dephillipsmichael) [reply BiomarinPKU/BiomarinPKU.xcodeproj/project.pbxproj]: I was able to successfully still deploy on my device with the new setup.

**Comment by dephillipsmichael** [comment]
Closing this one as your other PR https://github.com/Sage-Bionetworks/Biomarin-PKU-iOS/pull/27 has this too

**Pull request closed without merging**